django/db/backends/mysql/schema.py

```diff
@@ -230,6 +230,19 @@ def _alter_column_type_sql(
             model, old_field, new_field, new_type, old_collation, new_collation
         )
 
+    def _field_db_check(self, field, field_db_params):
+        if self.connection.mysql_is_mariadb and self.connection.mysql_version >= (
+            10,
+            5,
+            2,
+        ):
```

Comment:
☹️

:)

Reply:
not much I can do about it :shrug:

Reply:
I know, I checked :) — But since I saw it again, I had to comment. :)